packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -219,17 +225,17 @@ const SignupContainer = ({
 
   // TODO: In FXA-8305 - create option for LoadingSpinner to use these classes and use it throughout
   // settings since we use this set of classes often
-  const pageSpinner = (
```

Comment:
Why take this out of a `const`?

Reply:
Why render the control if it is not displayed? Also in the event you want to spy on whether or not the page spinner is rendered, which I didn't do but could have, always rendering would mess up that kind of test logic.

Reply:
I don't believe setting it to a `const` renders it, only if there is a `return`. Ideally this is pulled outside the component entirely though like with that referenced ticket.